cmd/private-org-sync/main.go

```diff
@@ -24,9 +26,10 @@ import (
 )
 
 type options struct {
-	configDir string
-	tokenPath string
-	targetOrg string
+	configDir     string
+	tokenPath     string
+	targetOrg     string
+	whitelistFile string
```

Comment:
Hm -- we're adding a new option that filters the set of config we operate on? Could we follow the pattern set up for `promotion.Options` and `config.Options` that expose their own `OperateOnCIOperatorConfigDir`?